int32 MIN_USERIO_OFFSET=128
int32 USERIO_ALIGNMENT=8

int32 offset
int32 size
uint8[] value

